tests/plugins/balloontoolbar/manual/dropdownhover.html

```diff
@@ -0,0 +1,44 @@
+<textarea id="editor1" cols="10" rows="10">
+	<p>Some text</p>
+	<p><strong>Click on me</strong></p>
+</textarea>
+
+<script>
+	// Currently balloon toolbar is not integrated with old IE browsers.
+	if ( CKEDITOR.env.ie && CKEDITOR.env.version < 9 ) {
+		bender.ignore();
+	}
+
+	CKEDITOR.disableAutoInline = true;
```

Comment:
There is no inline editor (or anything which would become one), so this line looks redundant.